draft-specifications/Model.md

```diff
@@ -395,9 +368,6 @@ Tagged values are equal or equivalent for the purposes of [Pass Invariant
 Equality](#pass-invariant-equality) if the tag and value are equivalent,
 transitively.
```

Comment:
Well, now that we've gotten rid of the unnecessary distinction between identity and equality, can we also dispense with "equivalent" and just speak of "equality"? What does talk of equivalence add to the meaning of this sentence?